Update README.md (#55)
canonical/jenkins-agent-operator: README.md

# Jenkins-agent Operator

[![CharmHub Badge](https://charmhub.io/jenkins-agent/badge.svg)](https://charmhub.io/jenkins-agent)
[![Publish to edge](https://github.com/canonical/jenkins-agent-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/jenkins-agent-operator/actions/workflows/publish_charm.yaml)
[![Promote charm](https://github.com/canonical/jenkins-agent-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/jenkins-agent-operator/actions/workflows/promote_charm.yaml)

A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing Jenkins agents on machines. It is configurable to integrate with a Jenkins charm deployed in another Juju model.

> This charm simplifies initial deployment and "day N" operations of Jenkins Agent on VMs and bare metal.

Reviewer: ```suggestion
This charm simplifies the deployment and operations of Jenkins Agent on VMs and bare metal.
```
To move away from "day N" jargon

As such, the charm makes it easy for those looking to take control of their own agents whilst keeping operations simple, and gives them the freedom to deploy on the platform of their choice.

For DevOps or SRE teams this charm will make operating Jenkins Agent simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing changes, and supports scaling out for enterprise deployments.

For information about how to deploy, integrate, and manage this charm, see the Official [jenkins-agent charm Documentation](https://charmhub.io/jenkins-agent/docs).

## Get started

> To begin, refer to the [Getting Started](https://charmhub.io/jenkins-agent/docs/tutorial-getting-started) tutorial for step-by-step instructions.

Reviewer: ```suggestion
To begin, refer to the [tutorial](https://charmhub.io/jenkins-agent/docs/tutorial-getting-started) for step-by-step instructions.
```
To try and move away from "Getting Started" or "quick guide" type of terminology

### Basic operations

#### Integrate with the Jenkins-k8S charm

See the [deploy and integrate machine agents](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#deploy-and-integrate-machine-agents-optional) section in the jenkins-k8s-operator documentation.

## Learn more

- [Read more](https://charmhub.io/jenkins-agent/docs)

> 

Reviewer: Is there an official website for Jenkins that is worth listing here?
Similarly, are there any developer docs for Jenkins?
Finally, could you add an item [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com) to this list?

Author: Not that I'm aware of, the agent binary is shipped with Jenkins and the charm is only wrapping a shell script to download it from Jenkins

- I added the troubleshooting link

## Project and community

The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.

* [Issues](https://github.com/canonical/jenkins-agent-operator/issues) <!--Link to GitHub issues (if applicable)-->

> * [Contributing](https://charmhub.io/jenkins-agent#contributing-to-this-documentation) <!--Link to any contribution guides--> 

Reviewer: Could you link https://github.com/canonical/jenkins-agent-operator/blob/main/CONTRIBUTING.md instead?

Author: Updated the link

For information about how to deploy, integrate, and manage this charm, see the Official [jenkins-agent charm Documentation](https://charmhub.io/jenkins-agent/docs).

## Get started

To begin, refer to the [tutorial](https://charmhub.io/jenkins-agent/docs/tutorial-getting-started) for step-by-step instructions.

### Basic operations

#### Integrate with the Jenkins-k8S charm

See the [deploy and integrate machine agents](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#deploy-and-integrate-machine-agents-optional) section in the jenkins-k8s-operator documentation.

## Learn more

- [Read more](https://charmhub.io/jenkins-agent/docs)
- [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)

## Project and community

The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.

* [Issues](https://github.com/canonical/jenkins-agent-operator/issues) <!--Link to GitHub issues (if applicable)-->
* [Contributing](https://github.com/canonical/jenkins-agent-operator/blob/main/CONTRIBUTING.md) <!--Link to any contribution guides--> 

> - [Matrix](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)

Reviewer: ```suggestion
* [Matrix](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
```
Rendering looks a little strange with mixing the `*` and `-`
